pkgs/development/python-modules/wirelesstagpy/default.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  requests,
+  pytestCheckHook,
+  requests-mock,
+}:
+
+buildPythonPackage rec {
+  pname = "wirelesstagpy";
+  version = "0.8.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "sergeymaysak";
+    repo = pname;
```

```suggestion
    repo = "wirelesstagpy";
```
do not rely on repo and program names matching